ViteMaDose/Helpers/Utils/UserDefaultsUtils.swift

```diff
@@ -37,18 +37,18 @@ extension UserDefaults {
     // MARK: Keys
 
     private enum Key: String {
-        case lastSelectedCountyCode
+        case lastSelectedDepartmentCode = "lastSelectedCountyCode"
```

Comment:
Should we consider migrating this old data in UserDefault to use renamed keys also ?

Reply:
This will actually change in #92 😬